packages/dev/package.json

```diff
@@ -9,11 +9,11 @@
     "serve": "serve public"
   },
   "dependencies": {
-    "@patternfly/patternfly": "^5.0.0",
-    "@patternfly/quickstarts": "^5.0.0-prerelease.2",
+    "@patternfly/patternfly": "^6.0.0-alpha.139",
```

Comment:
I think we want to get rid of the `^` here don't we? I feel like we've had issues with the core package and the `^` pulling in a wrong version.